### DISCUSSION TITLE
Alternative Package for Python?

### DISCUSSION THREAD

**Discussion by cool4wind** [charmbracelet/bubbletea#411] [General]
It's cool in Go, what about Python, any recommendations for Python developers?

**Comment by maaslalani** [comment]
Hey @cool4wind, check out https://github.com/Textualize/rich and https://github.com/Textualize/textual. I think they might be what you are looking for!

**Reply by bashbunni** [reply]
I've also heard of Textualize for Python!

**Reply by bitnom** [reply]
I'm surprised nobody has made a wrapper. Is there some complexity I'm overlooking?

**Reply by simonprovost** [reply]
Surprised too! I mean Charm libs are from another world! There has to be a wrapper someday 👀

Found that but not BubbleTea per se more `Gum`:
- https://github.com/idobarel/gum-python

Found that but given the structure of the repo and the non maintenance I would not ever try:
- https://github.com/utcq/GumWrapperPy
